__tests__/main.test.js

```diff
@@ -481,8 +481,8 @@ test('successfully runs the action with the select_label option', async () => {
             labels: []
           },
           {
-            number: 3,
-            title: 'Update dependency 3',
+            number: 4,
+            title: 'Update dependency 4',
```

Comment:
I _think_ the intention here was to increment the `number:` value for each branch created by the test, so I adjusted this to match that expectation.